src/utils/formatting.ts

```diff
@@ -99,3 +100,18 @@ export const formatTimeToHMS = (time: string) => {
     seconds
   )}s`;
 };
+
+export const formatMarketId = (marketId: string) => {
+  const timestamp = marketId.slice(0, 6);
+  const location = marketId.slice(6, 9);
+  const place = marketId.slice(9, 11);
+  const date = dayjs(0)
+    .add(+timestamp, "days")
+    .format("DD-MM-YYYY");
+  return `${date} ${location} Race ${place}`;
+};
+
+export const formatpropositionId = (propositionId: string) => {
+  const winningPropositionId = propositionId.slice(12, 14);
+  return `Horse ${winningPropositionId} win`;
```

Comment:
i think we should return just the number from this function rather than the full text

Reply:
This is the text in the ticket, but I understand your comment :)

Reply:
as a util function it should return only the number, and it can then be placed in a string literal in the component it's needed in. What happens when we want to use the proposition id and don't need the `Horse ... win` string? Or we want to account for other bets like place betting etc...